docs/jax.export.rst

```diff
@@ -0,0 +1,36 @@
+``jax.export`` module
+=====================
+
+.. automodule:: jax.export
```

Comment:
Do you plan to add the `jax.export` module as part of this PR? If not I think this will lead to errors

Reply:
The `jax.export` module was added in a previous PR.